semanticdb/metacp/src/main/scala/scala/meta/internal/scalacp/SymbolOps.scala

```diff
@@ -144,6 +148,60 @@ trait SymbolOps { self: Scalacp =>
   }
 
   implicit class XtensionSymbol(sym: Symbol) {
+    def isJavaDefined: Boolean = scalacp.isJavaDefined.get(sym) match {
+      case Some(cachedResult) =>
+        cachedResult
+      case _ =>
+        val result =
+          if (sym.isReallyPackage) false
```

Comment:
What does `really` mean here?

Reply:
Updated with comment and renamed to `isPackageAccordingToClasspath`
```scala
    // scalap Symbol.isPackage returns a constant false for external symbols so we have to
    // query the classpath to know which symbols are truly packages.
```